projects/ion/src/lib/utils/tests.ts

```diff
@@ -0,0 +1,7 @@
+export const ToHaveAttribute = (
+  element: HTMLElement,
+  attribute: string,
+  value: string
+): boolean => {
+  return element.getAttribute(attribute) === value;
```

Comment:
I think this isn't necessary anymore, we already can use jest-dom's `.toHaveAttribute()`.